server/github_test.go

```diff
@@ -406,7 +406,7 @@ func TestRateLimitTransport(t *testing.T) {
 		ghClient := server.NewGithubClientWithLimiter("testtoken", limit, 10, metricsMock)
 		_, _, err := ghClient.Git.GetRef(ctx, "ownerTest", "repoTest", "refTest")
 		require.Error(t, err)
-		require.Contains(t, err.Error(), "would exceed context deadline")
+		require.Contains(t, err.Error(), "context deadline")
```

Comment:
Actually this is not related to this ticket but it was failing on my local tests, I think `context deadline` would give enough hint for the error.